ui/src/hooks.ts

```diff
@@ -59,5 +59,9 @@ export function useConceptSet() {
 }
 
 export function useUrl() {
```

Comment:
Rename this to better describe what it actually does now. Something like `useUndoRedoURLs`.